rundeckapp/grails-app/views/menu/jobs.gsp

```diff
@@ -24,7 +24,7 @@
     <meta name="skipPrototypeJs" content="true"/>
     <g:set var="projectName" value="${params.project ?: request.project}"/>
     <g:set var="projectLabel" value="${session.frameworkLabels?session.frameworkLabels[projectName]:projectName}"/>
-    <g:set var="paginateJobs" value="${grailsApplication.config.rundeck.gui.paginatejobs}" />
+    <g:set var="paginateJobs" value="${grailsApplication.config.rundeck.gui.paginatejobs.enabled}" />
```

Comment:
If this property is set to false in the rundeck-config.properties it seems to still show the pagination controls.

Reply:
ok i'll check that

Reply:
should be fixed now